champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/assassin/axe/Blink.java

```diff
@@ -56,7 +56,7 @@ public String[] getDescription(int level) {
         return new String[]{
                 "Right click with a axe to activate.",
                 "",
-                "Instantly teleport forwards 15 Blocks.",
+                "Instantly teleport forwards " + ChatColor.GREEN + 15 + (3 * (level-1)) + ChatColor.GRAY + " Blocks.",
```

Comment:
should be (15 + (3 * (level-1)) otherwise it will just append 15 to the string

e.g. Instantly teleports forwards 153 blocks instead of Instantly teleports forwards 18 blocks